docs/azure/azure-how-to/instances/find-ubuntu-images.rst

```diff
@@ -358,17 +357,53 @@ Ubuntu CVM 22.04 LTS - Jammy Jellyfish
      - **URN**
    * - AMD64
      - Gen2
-     - ``Canonical:ubuntu-22_04-lts:cvm:latest``
+     - ``Canonical:ubuntu-24_04-lts:cvm:latest``
+
+Ubuntu Pro 22.04 LTS (Confidential Compute) - Jammy Jellyfish
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+| Quick start: `Ubuntu Pro 22.04 LTS (Confidential Compute) on Azure <https://portal.azure.com/#create/canonical.ubuntu-22_04-ltsubuntu-pro-cvm>`_
+| Quick start (China): `Ubuntu Pro 22.04 LTS (Confidential Compute) on Azure China <https://portal.azure.cn/#create/canonical.ubuntu-22_04-ltsubuntu-pro-cvm>`_
+
+.. list-table::
+   :widths: 10 12 78
+   :header-rows: 1
+
+   * - **Arch**
+     - **Hyper-V**
+     - **URN**
    * - AMD64
      - Gen2
      - ``Canonical:ubuntu-22_04-lts:ubuntu-pro-cvm:latest``
 
+Ubuntu 22.04 LTS (Confidential Compute) - Jammy Jellyfish
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+| Quick start: `Ubuntu Pro 22.04 LTS (Confidential Compute) on Azure <https://portal.azure.com/#create/canonical.ubuntu-22_04-ltscvm>`_
+| Quick start (China): `Ubuntu Pro 22.04 LTS (Confidential Compute) on Azure China <https://portal.azure.cn/#create/canonical.ubuntu-22_04-ltscvm>`_
+
+.. list-table::
+   :widths: 10 12 78
+   :header-rows: 1
+
+   * - **Arch**
+     - **Hyper-V**
+     - **URN**
+   * - AMD64
+     - Gen2
+     - ``Canonical:ubuntu-22_04-lts:cvm:latest``
+
 
 FIPS compliant offers
 ~~~~~~~~~~~~~~~~~~~~~
 
 To learn more about FIPS: `FIPS for Ubuntu <https://ubuntu.com/security/certifications/docs/fips>`_
 
+.. note::
+  | Starting with Ubuntu Pro FIPS 22.04 LTS, images validate against FIPS 140-3 and enable the ``fips-updates`` pro-service by default to ensure certified modules receive security updates. Prior releases validate against FIPS 140-2 and use strict ``fips`` certified modules as previously required by `FedRAMP <https://www.fedramp.gov/>`_.
+  |
+  | See this `Ubuntu blog post <https://ubuntu.com/blog/the-role-of-fips-140-3-in-the-latest-fedramp-guidance>`_ for more information about how ``fips-updates`` is more secure for your cloud environment.
```

Comment:
`Canonical blog post` instead of `Ubuntu blog post`.

Also, if you adopt the suggested wording above, consider appending this sentence to the preceding paragraph.